### PR TITLE
Change links to relative links using hugo shortcodes

### PR DESCRIPTION
**Pull request by wagnerpeer**
Partially adresses issues #45 and #106

### CODE REVIEW

**Comment by calculatinginfinity** [comment]
I could not review every single file but at least in this post the links are not working: 
* content/blog/2015-02-11-drew-in-hamburg.md 
* https://deploy-preview-117--dev-codefor-de.netlify.app/blog/drew-in-hamburg/

**Comment by wagnerpeer** [comment]
> I could not review every single file but at least in this post the links are not working:
> 
>     * content/blog/2015-02-11-drew-in-hamburg.md
> 
>     * https://deploy-preview-117--dev-codefor-de.netlify.app/blog/drew-in-hamburg/

Usage of bare HTML prevents correct processing of markdown formatting.

**Comment by ubergesundheit** [comment]
Blocked until all pages are there

File: content/blog/2019-10-17-CFG-Summit-FORUM-Offene-Stadt.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Um am FORUM Offene Stadt 2019 teilzunehmen bitten wir um Anmeldung unter [diesem
 
 **Am darauf folgenden Wochenende, Sa, 09. + So, 10. November findet erstmalig der Code for Germany Community Summit, beim Fablab St. Pauli in Hamburg, statt**. Ein in diesem Jahr ins Leben gerufenes Austausch- & Innovations Festival der Open Knowledge & Open Source Community, das von nun an jährlich stattfinden soll. Im Vordergrund steht der Austausch der ehrenamtlich tätigen Innovator*innen, Wissenschaftler*innen, Entwickler*innen, Policy-Spezialist*innen und Designer*innen  aus verschiedenen Städten in ganz Deutschland und die Weiterentwicklung der gemeinsamen Ideen und Ziele rund um offene Daten.
 
-Wenn ihr neben dem FORUM Offene Stadt auch am Code for Germany Summit teilnehmen möchtet, [meldet euch bitte hier an](https://codefor.de/summit/)!
+Wenn ihr neben dem FORUM Offene Stadt auch am Code for Germany Summit teilnehmen möchtet, [meldet euch bitte hier an]({{< ref "/summit" >}})!
```
Comment (ubergesundheit) [inline content/blog/2019-10-17-CFG-Summit-FORUM-Offene-Stadt.md:37]: Mh der `/summit` link in dem blog post scheint hugo ein bisschen schluckauf zu geben. Mein Vorschlag: 
```suggestion
Wenn ihr neben dem FORUM Offene Stadt auch am Code for Germany Summit teilnehmen möchtet, [meldet euch bitte hier an](/archiv/summit)!
```

**Pull request merged**